libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -227,28 +233,39 @@ void AP_Baro::calibrate(bool save)
    update the barometer calibration
    this updates the baro ground calibration to the current values. It
    can be used before arming to keep the baro well calibrated
+
+   The GPS is used to estimate and corect foour deviation from an ISA day.
```

Comment:
for our, yes. I see you have discovered my spelling terribleness :) (don't review the commit messages I'm sure they are even worse :D)